### PR TITLE
feat: 🎸 add elys testnet network

### PR DESCRIPTION
**Pull request by cosmic-vagabond**
This pull request adds configuration files for the Elys Network testnet. The first file, `assetlist.schema.json`, defines the native token of the network, including its symbol, base unit, and coingecko ID. The second file, `chain.schema.json`, defines various properties of the network, including its name, status, consensus type, and available APIs.

### CODE REVIEW

File: testnets/elystestnet/assetlist.json [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+      "denom_units": [
+        {
+          "denom": "uelys",
+          "exponent": 0,
+          "aliases": [
+            "microelys"
+          ]
+        },
+        {
+          "denom": "mscrt",
```
Comment (JeremyParish69) [inline testnets/elystestnet/assetlist.json:16]: I'm guessing this denom_unit is from a copy/paste error?

File: testnets/elystestnet/chain.json [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+      "type": "tendermint",
+      "version": "0.34"
+    },
+    "ibc_go_version": "v6.1.0",
+    "cosmwasm_enabled": false,
+    "genesis": {
+      "genesis_url": "https://snapshots.polkachu.com/testnet-genesis/elys/genesis.json"
+    },
+    "versions": [
+      {
```
Comment (JeremyParish69) [inline testnets/elystestnet/chain.json:42]: can we please rearrange versions from earliest to latest? (i.e, start with v0.2.3, then v0.3.1, then v0.4.0)

File: testnets/elystestnet/chain.json [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+    "compatible_versions": [
+      "v0.4.0"
+    ],
+    "cosmos_sdk_version": "v0.46",
+    "consensus": {
+      "type": "tendermint",
+      "version": "0.34"
+    },
+    "ibc_go_version": "v6.1.0",
+    "cosmwasm_enabled": false,
```
Comment (JeremyParish69) [inline testnets/elystestnet/chain.json:37]: can we please copy this into the latest version as well? (i.e., into the v0.4.0 object in the 'versions' array)

**Review by ollieottersync — CHANGES_REQUESTED** [review]
recommend this logo instead 
elys

File: testnets/elystestnet/assetlist.json (hunk 1 above)
Comment (ollieottersync) [reply testnets/elystestnet/assetlist.json:16]: @JeremyParish69 yes will correct that

Comment (cosmic-vagabond) [reply testnets/elystestnet/assetlist.json]: @JeremyParish69 solved

File: testnets/elystestnet/chain.json (hunk 2 above)
Comment (cosmic-vagabond) [reply testnets/elystestnet/chain.json:42]: @JeremyParish69 solved

File: testnets/elystestnet/chain.json (hunk 3 above)
Comment (cosmic-vagabond) [reply testnets/elystestnet/chain.json:37]: @JeremyParish69 solved

**Comment by cosmic-vagabond** [comment]
@JeremyParish69 thanks so much for your review, I fixed all the issues, could you review it again?

**Pull request merged**